detect_meteors.py

```diff
@@ -176,11 +182,52 @@ def detect_meteors(rf_dir, id_dir, noise_dir, output_dir,
         tmm.loadFromHdf5('/tmp/tmm.hdf5', skip_lowlevel=True)
     else:
         tmm.loadFromHdf5(skip_lowlevel=True)
-
-    for k, (tx, rx) in enumerate(data_generator(rfo, ido, no, tmm, s0, s1, rxch, txch)):
-        #FIXME call processing functions here
-        pass
-
+    
+    data_list = []
+    saved_data = np.zeros((59858, 1120))
+    pulse_data = data_generator(rfo, ido, no, tmm, s0, s1, rxch, txch)
+    times = []
+    for k, (tx, rx) in enumerate(pulse_data):
+
+        num_of_pulses = np.int((s1 - s0)/(rx.sample_rate*0.0267300605774))
+        # function that calculates snr values
+        def data_cal(array):
+            snr_vals = (np.abs(array.values)**2)/rx.noise_power
+            snr_point = np.unravel_index(np.argmax(snr_vals), (6879, 480))
+            max_snr = snr_vals[snr_point[0], snr_point[1]]
+            return max_snr, snr_point, snr_vals
+
+        mf_rx = freq_dft(tx, rx)
+        max_snr, snr_point, snr_vals = data_cal(mf_rx)
+        meteor_list = is_there_a_meteor(mf_rx, max_snr, snr_point, 1, 20533, 211200)
```

Comment:
Unidentified constants are confusing. As a minimum, it would be useful in this case to pass the values as keyword arguments instead of positional arguments, so people reading the code will know what they are meant to stand for. These frequency bounds are something that can change since they are one of the parameters that we would want to tweak. So, the thing to do here would be to include them somehow as keyword arguments to the detect_meteors function, and beyond that as command line arguments to the script. Since it is more intuitive to specify these bounds as a speed, that is how we should do it in the outer function. Then we can convert to frequency before passing them to this function.